support/isFunctionLengthConfigurable.js

```diff
@@ -0,0 +1,4 @@
+
+module.exports = function isFunctionLengthConfigurable(arg) {
+    return Object.getOwnPropertyDescriptor(function () { }, 'length').configurable;
+}
```

Comment:
this already exists on npm - https://npmjs.com/functions-have-names in the `functionsHaveConfigurableNames` property. let's just use that.